src/app/layout/WorkSpace/modules/PackCanvas/PackCanvas.tsx

```diff
@@ -40,6 +40,7 @@ const useStyles = makeStyles(({ bgPixel }) =>
       marginTop: (props: StyleProps) => `${props.height / -2}px`,
       transform: (props: StyleProps) =>
         `scale(${props.scale}) translate(${props.offsetX}px,${props.offsetY}px)`,
+      image-rendering: pixelated,
```

Comment:
CSS in JS needs to use `imageRendering: 'pixelated',` .

Reply:
oh sorry, i am not good with CSS and JS

Reply:
@RUSshy
Thank you for your PR, I have commit this optimization.   https://github.com/SilenceLeo/snowb-bmf/commit/29e831c4b2f346ab7024a447a6b9dd9b308d6508